pipelines/rj_cvl__osinfo_mongo/tasks.py

```
@@ -0,0 +1,642 @@
+                batch_data = db.fetch_batch(mongo_batch_size)
+                if not batch_data or len(batch_data) == 0:
+                    break
+                all_chunks.extend(batch_data)
+                chunk_count += len(batch_data)
+                if chunk_count % 100000 == 0:
+                    print(f"  Fetched {chunk_count:,} chunks so far...")
+
+            print(f"  Total chunks fetched: {len(all_chunks):,}")
+
+            if not all_chunks:
+                print(f"  No data found for this batch, skipping...")
```

Ruff (0.14.8)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)